src/secondStrategy.ts

```diff
@@ -0,0 +1,73 @@
+import {Position} from './position'
+ 
+export class SecondStrategy {
+    constructor() {}
+    public getNextTarget(pos: Position, triedMap: { [ pos: string] : boolean }, hitMap: { [ pos: string] : boolean }): Position  {
+        //first shoot at position that have 2 neighbours already hit
+        var iterPos = new Position("A", 1);
+        while(iterPos.row != "J" || iterPos.column != 10) {
+            var positionString = iterPos.getString();
+            if(!triedMap[positionString] && iterPos.hasTwoNeighbours(hitMap)){
+                return iterPos;
+            }
+            iterPos = iterPos.getNextPosition();
+        }
```

Comment:
Again, I'd be cutting out these strategy positions into separate methods and putting iterPos on the class, if only to cut the lenght of this thing.

(Much easier to read this if it's something like:

```
// reset iterPos
iterPos = new Position("A", 1);
shootAtPositionWith2AdjancentHits(iterPos, otherStuff);
shootAtPositionWith1AdjacentHit(iterPos, otherStuff);
etc.
```

But otherwise, very readable.